worker/src/test/java/gov/cms/ab2d/worker/processor/coverage/CoverageDriverTest.java

```diff
@@ -730,30 +747,6 @@ void availableCoverageWhenSinceContainsOnlySuccessful() {
         }
     }
 
-    @DisplayName("Create a coverage period and a mapping job for an eob job if any periods do not exist or have never" +
```

Comment:
Document why this was removed?

Reply:
Where best to document. I removed it because coverageperiod is no longer used in contracts.